src/client/chessboard/svg-components.tsx

```diff
@@ -0,0 +1,56 @@
+import { forwardRef } from "react";
+import { CustomSquareProps } from "react-chessboard/dist/chessboard/types";
+
+/**
+ * SVG of the highlight used to show a legal capture
+ */
+export const PieceSquare: React.FC<CustomSquareProps> = forwardRef(
```

Comment:
Since we aren't using the ref here, this should be a regular React component - declare it using the function keyword and add props and a prop interface if you need it.